### DISCUSSION TITLE
Support for FBQ50C7VEB

### DISCUSSION THREAD

**Discussion by ruimmsimoes** [revk/xESP32-Faikout#109] [Q&A]
Hello,
I am trying to figure out if the faikin board will be compatible with my Daikin FBQ50C7VEB.
This unit was manufactured in 2010 (don't know if this info helps).

I was looking at the connections diagram trying to find the S21 or X50A connectors, but haven't been able to find them.

Here is the diagram:
![Circuito AC Sala](https://github.com/revk/ESP32-Faikin/assets/78908626/c6733814-5d3f-42c2-81b9-606def127911)

Can anyone help me figure if this might be supported?

**Comment by ruimmsimoes** [comment]
Managed to take a picture of the board and it has a X50A connector.
I've ordered a Faikin board from amazon and will test to see if it is compatible as soon as I manage to create a cable.

**Reply by revk** [reply]
Cool, let me know how it goes.

Always improving the code as well.

**Reply by alexcf** [reply]
@ruimmsimoes did it work? :)

**Reply by ruimmsimoes** [reply]
Yes, it is working.
